src/Components/AddMedia/AddMedia.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useRef } from "react";
+import icon from "./photo.svg";
+import styles from "./AddMedia.module.css";
+import { Media } from "..";
+
+type Props = {
+  addFile: (newMedia: Media[]) => void;
+  mediaElement: (addFiles: Media[], file: File) => void;
```

Comment:
can this be renamed so it's clear it's a function?
Functions should start with a verb, such as `addMediaElement` or `createMediaElement` etc. (see Naming things: https://docs.canva.tech/style-guides/universal/#use-a-verb-phrase-for-an-action)

Check other fns too